src/styles/sidebar/components/group-section.scss

```diff
@@ -0,0 +1,47 @@
+/* The groups section. */
+
+.group-section {
+  .dropdown-menu {
+    width: $group-list-width;
+    max-height: 500px;  // fallback for browsers lacking support for vh/calc
```

Comment:
All of the browsers we support have support for both `calc` and viewport units. See https://caniuse.com/#search=calc and https://caniuse.com/#search=viewport%20units, so we might not need the fallback here. There are mentions of bugs in some older browsers though. I haven't looked in detail to see if that affects us.